dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dataset/DatasetThumbnailService.java

```
@@ -373,7 +328,7 @@ public boolean isDatasetLogoPresent(Dataset dataset) {
 
         try {
             storageIO = dataAccess.getStorageIO(dataset);
-            return storageIO.isAuxObjectCached(datasetLogoThumbnail + thumb48addedByImageThumbConverter);
+            return storageIO.isAuxObjectCached(datasetLogoThumbnail48);
         } catch (IOException ioex) {
         }
```

Empty catch – we should log something at least.